adapter/internal/oasparser/operator/operator.go

```diff
@@ -145,11 +96,11 @@ func GetXWso2LabelsWebSocket(webSocketAPIDef model.MgwSwagger) []string {
 }
 
 /*
-GetMgwSwaggerWebSocket returns a MgwSwagger for the web socket APIs
+GetMgwSwaggerFromAPIYaml returns a MgwSwagger for the APIs using the API.Yaml file.
 */
-func GetMgwSwaggerWebSocket(apiData model.APIYaml) (model.MgwSwagger, error) {
+func GetMgwSwaggerFromAPIYaml(apiData model.APIYaml, apiType string) (model.MgwSwagger, error) {
```

Comment:
Shall we remove this method and directly call SetSwaggerFromAPIYaml from other places?